packages/sitecore-jss/src/site/graphql-siteinfo-service.ts

```diff
@@ -0,0 +1,137 @@
+import { GraphQLClient, GraphQLRequestClient } from '../graphql';
+import debug from 'debug';
+import { CacheClient, CacheOptions, MemoryCacheClient } from '../cache-client';
+
+const siteGroupingTemplate = 'EDA823FC-BC7E-4EF6-B498-CD09EC6FDAEF';
+
+const defaultQuery = /* GraphQL */ `
+  {
+    search(
+      where: { name: "_template", value: "${siteGroupingTemplate}", operator: EQ }
+    ) {
+      results {
+        ... on SiteDefinition {
+          name
+          hostName {
+            value
+          }
+          virtualFolder {
+            value
+          }
+          language: language_f19277fe1b854b0a8c265e74d766b3a3 {
```

Comment:
Should be able to use "field" to access this (vs the auto-gen'd strong type which wouldn't work elsewhere)
```suggestion
          language: field(name: "Language") {
```